README.md

```diff
@@ -10,7 +10,7 @@ CI status (Linux, macOS and Windows): [![Install and Test][1]][2]
 # CURRENT STATE
 
 A major refactoring of large parts of the codebase is currently in progress. It is expected that functionality on `main`
-is severely limited during this process. Please check out
+is limited during this process. Please check out
```

Comment:
Given the fact that we are now at a state where the functionality isn't really limited (probably except for the tag-value parser), I am wondering if we should enforce users to create issues here. This statement could make users think that all errors or misfunctionality is expected. What do you think?

Reply:
We should rather encourage than enforce them, but I agree :D

Reply:
Done, please check the new version! :)